src/alerts/Alerts.ts

```diff
@@ -1,18 +1,27 @@
+import { IconId } from '@terra-ui-packages/components';
 import _ from 'lodash/fp';
-import { Fragment, useEffect, useState } from 'react';
+import { CSSProperties, Fragment, ReactNode, useEffect, useState } from 'react';
 import { div, h, li, p, span, ul } from 'react-hyperscript-helpers';
 import { Clickable, Link } from 'src/components/common';
 import { icon } from 'src/components/icons';
 import PopupTrigger from 'src/components/PopupTrigger';
 import colors from 'src/libs/colors';
 import { useLinkExpirationAlerts } from 'src/libs/link-expiration-alerts';
 import { usePrevious } from 'src/libs/react-utils';
-import { useServiceAlerts } from 'src/libs/service-alerts';
 import { useTermsOfServiceAlerts } from 'src/libs/terms-of-service-alerts';
 import * as Utils from 'src/libs/utils';
 
-const Alert = ({ alert }) => {
-  const { title, message, link, linkTitle, severity } = alert;
+import { Alert as AlertT } from './Alert';
+import { useServiceAlerts } from './service-alerts';
+
+interface AlertProps {
+  alert: AlertT;
+}
+
+const Alert = (props: AlertProps): ReactNode => {
+  const {
+    alert: { title, message, link, linkTitle, severity },
```

Comment:
Non-blocking opt: The pattern here where alert is wrapped in an additional object seems a bit strange to me - should those be lifted up one level?

Reply:
Conceptually, the `Alert` component is taking one prop: a single `Alert` object.

The `Alert` component _could_ take all of the `Alert` object's properties as individual props, but I think keeping the alert as a single object makes more sense.